package.json

```diff
@@ -1,41 +1,47 @@
 {
-  "name": "@financial-times/n-url-management-api-read-client",
-  "version": "0.0.0",
-  "main": "main",
-  "dependencies": {
-    "@dotcom-reliability-kit/logger": "^2.2.7",
-    "aws-sdk": "^2.2.33",
-    "denodeify": "^1.2.1"
-  },
-  "devDependencies": {
-    "@financial-times/n-gage": "^8.3.2",
-    "chai": "^3.5.0",
-    "check-engine": "^1.10.1",
-    "eslint": "^1.10.3",
-    "istanbul": "^0.4.2",
-    "lintspaces-cli": "^0.6.0",
-    "mocha": "^5.0.0",
-    "mockery": "^1.4.0",
-    "npm-prepublish": "^1.2.1",
-    "proxyquire": "^1.7.4",
-    "sinon": "^1.17.3",
-    "snyk": "^1.168.0"
-  },
-  "scripts": {
-    "prepare": "npx snyk protect || npx snyk protect -d || true"
-  },
-  "engines": {
-    "node": "16.x || 18.x",
-    "npm": "7.x || 8.x || 9.x"
-  },
-  "husky": {
-    "hooks": {
-      "commit-msg": "node_modules/.bin/secret-squirrel-commitmsg",
-      "pre-commit": "node_modules/.bin/secret-squirrel",
-      "pre-push": "make verify -j3"
+    "name": "@financial-times/n-url-management-api-read-client",
+    "version": "0.0.0",
+    "main": "main",
+    "dependencies": {
+        "@dotcom-reliability-kit/logger": "^2.2.7",
+        "aws-sdk": "^2.2.33",
+        "denodeify": "^1.2.1"
+    },
+    "devDependencies": {
+        "chai": "^3.5.0",
+        "check-engine": "^1.10.1",
+        "eslint": "^7.32.0",
+        "istanbul": "^0.4.2",
+        "lintspaces-cli": "^0.6.0",
+        "mocha": "^6.2.3",
+        "mockery": "^1.4.0",
+        "npm-prepublish": "^1.2.1",
+        "proxyquire": "^1.7.4",
+        "sinon": "^1.17.3",
+        "snyk": "^1.168.0",
+        "dotcom-tool-kit": "^3.4.0",
+        "@dotcom-tool-kit/component": "^4.1.0",
+        "@dotcom-tool-kit/mocha": "^3.2.0",
+        "@dotcom-tool-kit/eslint": "^3.2.0",
+        "@financial-times/eslint-config-next": "^7.1.0"
+    },
+    "scripts": {
+        "prepare": "npx snyk protect || npx snyk protect -d || true",
+        "build": "dotcom-tool-kit build:local",
+        "test": "dotcom-tool-kit test:local",
+        "start": "dotcom-tool-kit run:local"
+    },
+    "engines": {
+        "node": "16.x || 18.x",
+        "npm": "7.x || 8.x || 9.x"
+    },
+    "husky": {
+        "hooks": {
+            "commit-msg": "dotcom-tool-kit git:commitmsg",
+            "pre-commit": "dotcom-tool-kit git:precommit"
```

Comment:
**question**: did you initially install the `lint-staged` plugin too? I'm not sure why Tool Kit added these hooks when `husky-npm` shouldn't be present 🤔